tests/integration_tests/test_examples.py

```diff
@@ -151,3 +147,48 @@ def test_pearl_ml1_push():
         '--embedding_mini_batch_size', '2', '--max_path_length', '1'
     ],
                           check=False).returncode == 0
+
+
+@pytest.mark.nightly
+@pytest.mark.no_cover
+@pytest.mark.timeout(120)
+def test_maml_ml10():
+    """Test maml_trpo_ml10.py"""
+    assert subprocess.run([
+        EXAMPLES_ROOT_DIR / 'torch/maml_trpo_ml10.py', '--epochs', '1',
```

Comment:
We use this code to make it run for only 1 epoch
```
env = os.environ.copy()
env['GARAGE_EXAMPLE_TEST_N_EPOCHS'] = '1'
```
This is in line with the philosophy of having to modify examples as least as possible for the sake of tests